python/route_guide/.gitignore

```diff
@@ -0,0 +1 @@
+*.pyc
```

Comment:
This may not be necessary or appropriate if the Python interpreter is invoked with "-B". Up to you if you want to keep it?